enterprise_subsidy/apps/subsidy/models.py

```diff
@@ -378,5 +442,8 @@ def is_entitled(self, learner_id, content_key):
 
 class LicenseRequestAccessPolicy(SubscriptionAccessPolicy):
```

Comment:
this code was part of the example code but it is meant for the subsidy access policy API, we can remove this.